unicorecmsmama/templates/base.pt

```diff
@@ -26,7 +26,7 @@
 
     <div id="header">
         <a href="/">
-          <img src="/static/img/logo-bc.png?v=2" border="0" />
+          <img src="/static/img/baby_h_rgb_5c.png?v=2" border="0" />
```

Comment:
Should we keep the name of the file `logo-bc.png` and just update `v=2` to `v=3`?